CONTRIBUTING.md

```diff
@@ -55,6 +55,7 @@ Nobody is perfect, and sometimes we mess things up. That said, here are some goo
 * Squash your commits when making revisions after a patch review.
 * Add your personal copyright line to files when making substantive changes. (Optional but encouraged!)
 * Check the spelling of your code, comments and commit messages.
+* If you have images that go along with your code, remember to do ` optipng -strip all ` in order to save a couple more bytes. You'll be glad you did.
```

Comment:
```suggestion
* If you have images that go along with your code, run `optipng -strip all` on them to optimize & strip away useless metadata - this can reduce file size from multiple kilobytes to a couple hundred bytes.
```

Reply:
Suggested change implemented.  :^)